src/components/Button/Button.vue

```diff
@@ -256,12 +260,9 @@ export default {
 		type: {
 			type: String,
 			validator(value) {
-				// Disallowing secondary until we solve theming and
-				// accessibility issues.
-				// https://github.com/nextcloud/nextcloud-vue/issues/2538
-				return ['primary', 'tertiary', 'error', 'warning', 'success'].indexOf(value) !== -1
+				return ['primary', 'secondary', 'tertiary', 'error', 'warning', 'success'].indexOf(value) !== -1
 			},
-			default: '',
+			required: true,
```

Comment:
Why not accepting the empty type? The default button shouldn't require specifying a type IMO

Reply:
Because then different classes are applied as discussed last week:
```js
[`button-vue--vue-${this.type}`]: this.type,
```

Or would `default: 'secondary'` work already?